File: Com.Fubei.Api.OpenApi/Biz/AgentApi/Model/Request/ACreateFixedQrCodeParam.cs
using System;
using System.Collections.Generic;
using System.Text;
using Com.Illuminati.Galileo.Model;
using Newtonsoft.Json;

namespace Com.Fubei.Api.OpenApi.Biz.AgentApi.Model.Request
{
    /// <summary>
    /// 聚合码支付
    /// http://docs.51fubei.com/agent-api/payment/qrCodePay.html
    /// </summary>
    public class ACreateFixedQrCodeParam : BaseEntity
    {
        [JsonProperty("merchant_id")]
        public int? MerchantId { get; set; }

        [JsonProperty("merchant_order_sn")]
        public string MerchantOrderSn { get; set; }

        [JsonProperty("total_amount")]
        public decimal TotalAmount { get; set; }

        [JsonProperty("store_id")]
        public int StoreId { get; set; }

        [JsonProperty("sub_appid")]
        public string SubAppId { get; set; }

        [JsonProperty("user_id")]
        public string UserId { get; set; }

        [JsonProperty("goods_tag")]
        public string GoodsTag { get; set; }

        [JsonProperty("detail")]
        public GoodsDetailEntity Detail { get; set; }

        [JsonProperty("device_no")]
        public string DeviceNo { get; set; }

        [JsonProperty("body")]
        public string Body { get; set; }

        [JsonProperty("attach")]
        public string Attach { get; set; }

        [JsonProperty("timeout_express")]
        public string TimeoutExpress { get; set; }

        [JsonProperty("notify_url")]
        public string NotifyUrl { get; set; }

        [JsonProperty("alipay_extend_params")]
        public AlipayExtendParamsEntity AlipayExtendParams { get; set; }

        [JsonProperty("expired_time")]
        public int ExpiredTime { get; set; } = 600;
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/AgentApi/Model/Request/ACreateFixedQrCodeParam.vb
Imports Com.Illuminati.Galileo.Model
Imports Newtonsoft.Json

Namespace Biz.V2.Model.Request
    
    ' 定额码支付
    ' http://docs.51fubei.com/agent-api/payment/orderQrPay.html
    Public Class ACreateFixedQrCodeParam
        Inherits BaseEntity

        <JsonProperty("merchant_id")>
        Public Property MerchantId As Integer?

        <JsonProperty("merchant_order_sn")>
        Public Property MerchantOrderSn As String

        <JsonProperty("total_amount")>
        Public Property TotalAmount As Decimal

        <JsonProperty("store_id")>
        Public Property StoreId As Integer

        <JsonProperty("sub_appid")>
        Public Property SubAppId As String

        <JsonProperty("user_id")>
        Public Property UserId As String

        <JsonProperty("goods_tag")>
        Public Property GoodsTag As String

        <JsonProperty("detail")>
        Public Property Detail As GoodsDetailEntity

        <JsonProperty("device_no")>
        Public Property DeviceNo As String

        <JsonProperty("body")>
        Public Property Body As String

        <JsonProperty("attach")>
        Public Property Attach As String

        <JsonProperty("timeout_express")>
        Public Property TimeoutExpress As String

        <JsonProperty("notify_url")>
        Public Property NotifyUrl As String

        <JsonProperty("alipay_extend_params")>
        Public Property AlipayExtendParams As AlipayExtendParamsEntity

        <JsonProperty("expired_time")>
        Public Property ExpiredTime As Integer = 600

    End Class
End Namespace
